src/pybind/mgr/prometheus/module.py

```diff
@@ -29,22 +29,6 @@
 
 DEFAULT_PORT = 9283
 
-# When the CherryPy server in 3.2.2 (and later) starts it attempts to verify
-# that the ports its listening on are in fact bound. When using the any address
-# "::" it tries both ipv4 and ipv6, and in some environments (e.g. kubernetes)
-# ipv6 isn't yet configured / supported and CherryPy throws an uncaught
-# exception.
-if cherrypy is not None:
-    Version = version.Version
-    v = Version(cherrypy.__version__)
-    # the issue was fixed in 3.2.3. it's present in 3.2.2 (current version on
-    # centos:7) and back to at least 3.0.0.
-    if Version("3.1.2") <= v < Version("3.2.3"):
-        # https://github.com/cherrypy/cherrypy/issues/1100
-        from cherrypy.process import servers
-        servers.wait_for_occupied_port = lambda host, port: None
```

Comment:
some flake8 complaints about this causing make check to fail
```
prometheus/module.py:11:1: F401 'packaging.version' imported but unused
prometheus/module.py:33:1: E302 expected 2 blank lines, found 1
prometheus/module.py:991:19: F811 redefinition of unused 'version' from line 11
prometheus/module.py:1053:19: F811 redefinition of unused 'version' from line 11
prometheus/module.py:1298:27: F811 redefinition of unused 'version' from line 11
1     E302 expected 2 blank lines, found 1
1     F401 'packaging.version' imported but unused
3     F811 redefinition of unused 'version' from line 11
```

Reply:
fixed it.